service/src/main/kotlin/io/provenance/explorer/service/ValidatorService.kt

```diff
@@ -407,6 +406,11 @@ class ValidatorService(
         )
     }
 
+    fun getValidatorMarketRateAvg(address: String, txCount: Int) =
+        ValidatorMarketRateRecord.getValidatorRateForBlockCount(address, txCount)
+            .map { it.marketRate }
+            .let { list -> MarketRateAvg(list.size, list.minOrNull()!!, list.maxOrNull()!!, list.average()) }
```

Comment:
This seems weird to call a method that can return null and then !! it

Reply:
The extension I would use is deprecated in favor of that one that currently being used
```

@Deprecated("Use minOrNull instead.", ReplaceWith("this.minOrNull()"))
@DeprecatedSinceKotlin(warningSince = "1.4", errorSince = "1.5")
public fun <T : Comparable<T>> Iterable<T>.min(): T? {
    return minOrNull()
}
```